frontend/layouts/main.tsx

```diff
@@ -16,11 +16,6 @@ const Main: React.FC<Props> = ({ children, headTitle, title }) => {
           {headTitle ?? "DB Cost | RDS & Cloud SQL Instance Pricing Sheet"}
         </title>
         <link rel="icon" href="/favicon.ico" />
-        <script
```

Comment:
Do we need to remove plausible? And we should remove it in a separate PR.

Reply:
Sorry, there is a `_document.tsx` with plausible.

Reply:
We're not removing it. Just moved to another place: https://github.com/bytebase/dbcost/blob/36b78c74808dd5634efa73da7bf43ddc0cffb6e3/frontend/pages/_document.tsx#L8